awpy/utils.py

```diff
@@ -1,16 +1,34 @@
-""" Util functions for csgo package
-"""
+"""Util functions for csgo package."""
 
 import re
 import subprocess
+from typing import Any
+
 import pandas as pd
+
 from awpy.types import Area
 
 
 class AutoVivification(dict):
```

Comment:
is defaultdict something we could use here?

Reply:
Yeah, i think think the Autovivification is not even used anywhere. I have a function that return a nested default dict like that already in the nav module.

https://github.com/pnxenopoulos/awpy/blob/main/awpy/analytics/nav.py#L370

I dont know if you want me to just delte the part here since it is currently only being used in nav or if i should replace this part with the definition from nav?